utils/avd_recreate.sh

```diff
@@ -15,7 +13,7 @@ if [[ "$(uname -m)" == "arm64" ]]; then
     ANDROID_ARCH="arm64-v8a"
 fi
 
-${ANDROID_HOME}/tools/bin/avdmanager delete avd --name android33 || true
-echo "y" | ${ANDROID_HOME}/tools/bin/sdkmanager --install "system-images;android-33;google_apis;$ANDROID_ARCH"
-echo "no" | ${ANDROID_HOME}/tools/bin/avdmanager create avd --force --name android33 --device "Nexus 5X" --package "system-images;android-33;google_apis;$ANDROID_ARCH"
+${ANDROID_HOME}/cmdline-tools/latest/bin/avdmanager delete avd --name android33 || true
```

Comment:
In more recent versions the paths have changed. Upstream (on GitHub Actions) and on local installations using `avd_install.sh`.